app/presenters/publishing_api/html_attachment_presenter.rb

```diff
@@ -75,7 +75,7 @@ def parent
     end
 
     def parent_content_ids
-      ::Edition.joins(:document).where(editions: { id: item.attachable_id }).pluck(:content_id)
+      [parent.document.content_id]
```

Comment:
👍 this definitely makes the code more readable and contained....I notice this is sort of reverting a change made to improve performance in [PR 2612](https://github.com/alphagov/whitehall/pull/2612). Do you think memoising  `item.attachable` in `def parent` would be a good idea? That might help maintain performance but also keep the more readable code.

Reply:
🤔 I'm not sure that would make much difference to do that; given that `attachable` is an activerecord association it'll be memoized anyway.

The note in the commit suggests it's done to avoid fully instantiating the associated `document`, which I think makes sense.  I'll do a few checks and see if it's still an issue, if so I'll find another way.

Reply:
OK I've looked into this, and while this does add back an extra request for the document, there's no way to avoid this now that HTML attachments can be further than one step from an edition, without serious refactoring.

Reply:
The document load is also very quick, so I don't think it's really a problem.  Avoiding loading all the organisations is the big gain, and we keep that here.

Reply:
👍awesome thanks for investigating